src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common_std/xml/AvoidConcatInAppend.xml

```diff
@@ -21,7 +21,10 @@ public class Foo {
 
     private void concatNumericIsBad() {
         StringBuilder sb = new StringBuilder();
-        sb.append(1 + 2); // bad
+        String bar = "bar";
+        sb.append(1 + 2 + "foo"); // bad
```

Comment:
in mijn beleving dus goed (line 25)

Reply:
updated (simplified) rule and unit tests